modules/ROOT/pages/provisioning-ibmcloud.adoc

```diff
@@ -1,6 +1,8 @@
 = Provisioning Fedora CoreOS on IBM Cloud
 
-This guide shows how to provision new Fedora CoreOS (FCOS) instances in IBM Cloud VPC Generation 2 for either the `x86_64` or `s390x` architectures.
+This guide shows how to provision new Fedora CoreOS (FCOS) instances in IBM Cloud for either the `x86_64` or `s390x` architectures.
```

Comment:
For the record, [this post](https://www.ibm.com/cloud/blog/announcements/start-your-vpc-gen1-to-vpc-gen2-migration) says that VPC Generation 1 EOLed in 2021, so we can indeed drop the `Generation 2` suffix.